src/icons/CloseChatWindowIcon.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+export default function CloseChatWindowIcon() {
```

Comment:
I think this icon can be named a little more generically. It's just an "X", so I don't think it is specific to the ChatWindow component. What do you think about `CloseIcon`?

Reply:
That makes more sense so that we can reuse it in the future if need be 👍